ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -884,3 +942,33 @@ mergeListsBy cmp all_lists = merge_lists all_lists
         []     -> []
         [xs]   -> xs
         lists' -> merge_lists lists'
+
+
+getCompletionPrefix :: (Monad m) => Position -> VFS.VirtualFile -> m (Maybe PosPrefixInfo)
+getCompletionPrefix pos@(Position l c) (VFS.VirtualFile _ _ ropetext) =
```

Comment:
Is this from Language.LSP.VFS? What have you changed?

Reply:
It doesn't really belong there, so moving it here is good (I'd like to delete it from `lsp` afterwards).